custom-typings/path-is-absolute.d.ts

```diff
@@ -0,0 +1,5 @@
+
+declare module "path-is-absolute" {
+    function pathIsAbsolute(path: string): boolean;
+    export = pathIsAbsolute;
```

Comment:
can you add a line here: `namespace pathIsAbsolute {}`. This will let us import it with ES6 syntax